app/uploaders/banner_image_uploader.rb

```diff
@@ -0,0 +1,8 @@
+class BannerImageUploader < CarrierWave::Uploader::Base
+  include CarrierWave::Compatibility::Paperclip
```

Comment:
@tdonohue yeah, we can expect to have multiple Sites per application instance. The storage question is a good one, and relates to https://github.com/projecthydra-labs/lerna/issues/149. You could potentially punt on that for now, and then take on 149 as a follow-on issue.